questions.md

```diff
@@ -0,0 +1,3 @@
+# Questions
+
+* What happens if there is a conflict in the pull request?
```

Comment:
👋 Hi @brianehenyo ! If there is a conflict in the pull request GitHub will notify you. I that case you can resolve the issue in GitHub, which is not recommended, or you can resolve it locally on your machine. After you have resolved the merge conflict and merged your work, GitHub will recognize this and immediately close the Pull Request for you.